css-transforms-2/Overview.bs

```diff
@@ -1106,6 +1106,65 @@ The transform functions <<matrix()>>, ''matrix3d()'' and ''perspective()'' get c
 
 For interpolations with the primitive ''rotate3d()'', the direction vectors of the transform functions get normalized first. If the normalized vectors are equal, the rotation angle gets interpolated numerically. Otherwise the transform functions get converted into 4x4 matrices first and interpolated as defined in section <a href="#matrix-interpolation">Interpolation of Matrices</a> afterwards.
 
+Addition and accumulation of transform lists {#combining-transform-lists}
+============================================
+
+<a lt="value addition">Addition</a> of transform lists is
+defined as <a spec=infra>list</a> concatenation
```

Comment:
You'll be using these a lot, I recommend just putting these terms into `link-defaults` for the spec.

Reply:
I looked up the bikeshed docs, but I couldn't work out how to setup an alias with `link-defaults` (from "value addition" → "addition"). I've done it with `anchors` before, but not `link-defaults`.

Reply:
Oh, this comment was referring to the `list` link -- I see! Sorry I assumed it was the first link.